site/content/entry/yuzu-progress-report-oct-2020/index.md

```diff
@@ -0,0 +1,205 @@
+- [Fixed crashes when trying to launch subsequent games.](https://github.com/yuzu-emu/yuzu/pull/4832)
+- [Improved GPU synchronization.](https://github.com/yuzu-emu/yuzu/pull/4869). 
+
+The last one also improves stability in games like `Luigi's Mansion 3`. Only thing missing to get `Super Mario 3D All-Stars` rendering is the `Texture Cache Rewrite`.
+
+## NVDEC improvements
+
+Work on video decoding continues, thanks to [epicboy](https://github.com/ameerj). Some games like `Onimusha: Warlords` or `Ni no Kuni`had an incompatible `BGRA8` format in our first NVDEC implementation. Now they render correctly.
+
+{{< imgs
+    "./ninokuni.png| Working menu rendering (Ni no Kuni)"
+  >}}
+  
+Additionally, games like `FINAL FANTASY XII THE ZODIAC AGE`, `The Legend of Zelda: Link's Awakening` or `Onimusha: Warlords` had problems on memory addressing, causing poor performance and glitches. Consider this as also solved.
+
+{{< imgs
+    "./onimusha.png| Happy moments (Onimusha: Warlords)"
+  >}}
+  
+And last but not least, NVDEC is now in Mainline too! Anyone can now enjoy they in-game videos in yuzu. As tradition for all merged Pull Requests, [Lioncache](https://github.com/lioncash) didn't take long to [clean it up.](https://github.com/yuzu-emu/yuzu/pull/4837)
+  
+## Non-mythological input changes
+
+We have some great changes, work of [german77](https://github.com/german77). Thank you!
+
+To start with, now [buttons that are impossible to use on certain input configurations are now blocked](https://github.com/yuzu-emu/yuzu/pull/4742). For example, if an emulated Pro Controller is changed to a left Joy-Con, the right stick, A, X, Y and Z will no longer be accessible to the games. No cheating!
+
+Up next, some devices have access to an accelerometer, but lack a gyroscope. While this setup is basically useless for full motion, it can be used to detect simple shaking motions, [so we don't see why not add support for that.](https://github.com/yuzu-emu/yuzu/pull/4757)
+
+And finally, for our traditional fighters out there, improvements in GC adapters. Now [adapters can be hotplugged, basic rumble support was added and compatibility with more vendors was improved.](https://github.com/yuzu-emu/yuzu/pull/4781)
+
+A new challenger approaches! [FrogTheFrog](https://github.com/FrogTheFrog) tuned up [the shake values](https://github.com/yuzu-emu/yuzu/pull/4727) to improve compatibility with the rare `Steam Controller`. Wouldn't having Portal and Portal 2 on Switch be awesome?
+
+## Bug fixes and improvements
+
+[bunnei](https://github.com/bunnei) [changed the values for `Multicore`, `Asynchronous GPU` and `Assembly Shaders`](https://github.com/yuzu-emu/yuzu/pull/4805) to enabled by default. This will help provide a much better experience out of the box to all users that haven't changed those values already. Some other advanced settings like `Asynchronous shaders` are not altered by this change as they still require further testing and bug-fixing.
```

Comment:
```suggestion
[bunnei](https://github.com/bunnei) [changed the values for `Multicore`, `Asynchronous GPU` and `Assembly Shaders`](https://github.com/yuzu-emu/yuzu/pull/4805) to be enabled by default. This will help provide a much better experience out of the box to all users that haven't changed those values already. Some other advanced settings like `Asynchronous shaders` are not altered by this change as they still require further testing and bug-fixing.
```